zebra/zebra_rib.c

```diff
@@ -1264,9 +1248,14 @@ static void rib_process(struct route_node *rn)
 	struct zebra_vrf *zvrf = NULL;
 	struct vrf *vrf;
 	struct route_entry *proto_re_changed = NULL;
-
+	struct rib_table_info *info = NULL;
 	vrf_id_t vrf_id = VRF_UNKNOWN;
 
+	if (IS_ZEBRA_DEBUG_RIB || IS_ZEBRA_DEBUG_RIB_DETAILED) {
+		info = srcdest_rnode_table_info(rn);
+		assert(info);
+	}
+
```

Comment:
I don't think info can be null, but SA isn't smart enough to realize that:

https://ci1.netdef.org/browse/FRR-PULLREQ3-STATICANALYZER-5787

However, to avoid adding a false positive, and see we only care about what safi we have below, I suggest to pull that value here and use it below without using `info` below. 
```
        safi_t safi = -1;

	if (IS_ZEBRA_DEBUG_RIB || IS_ZEBRA_DEBUG_RIB_DETAILED) {
	        struct rib_table_info *info;
		info = srcdest_rnode_table_info(rn);
		if (info)
                   safi = info->safi
	}
```
use `safi` below instead of `info->safi`